llvm/lib/Target/AArch64/AArch64InstrInfo.cpp

```diff
@@ -9461,6 +9462,94 @@ bool AArch64InstrInfo::isReallyTriviallyReMaterializable(
   return TargetInstrInfo::isReallyTriviallyReMaterializable(MI);
 }
 
+MachineBasicBlock::iterator
+AArch64InstrInfo::probedStackAlloc(MachineBasicBlock::iterator MBBI,
```

Comment:
I think it would be useful for this function to have some comments that explain what it emits, so it can be understood isolated from the rest of this patch.

Reply:
I've left a comment at the declaration in `AArch64InstrInfo.h`